.github/workflows/selfdrive_tests.yaml

```diff
@@ -174,7 +174,7 @@ jobs:
       timeout-minutes: ${{ ((steps.restore-scons-cache.outputs.cache-hit == 'true') && 10 || 30) }} # allow more time when we missed the scons cache
       run: ${{ env.RUN }} "scons -j$(nproc)"
     - name: Run unit tests
-      timeout-minutes: 1
+      timeout-minutes: ${{ contains(runner.name, 'nsc') && 1 || 5 }}
```

Comment:
let's just make them like 20 minutes for actions? I want to be strict for what runs on our repos, but we don't want to deal with that for actions